tools/schemacode/schema.py

```diff
@@ -322,3 +364,151 @@ def make_entity_table(schema, tablefmt="github", **kwargs):
     # Print it as markdown
     table_str = tabulate(table, headers="keys", tablefmt=tablefmt)
     return table_str
+
+
+def _get_link(string):
+    dict_ = {
+        "array": "[array](https://www.w3schools.com/js/js_json_arrays.asp)",
+        "arrays": "[arrays](https://www.w3schools.com/js/js_json_arrays.asp)",
+        "string": (
+            "[string](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "strings": (
+            "[strings](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "number": (
+            "[number](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "numbers": (
+            "[numbers](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "object": "[object](https://www.json.org/json-en.html)",
+        "objects": "[objects](https://www.json.org/json-en.html)",
+        "integer": (
+            "[integer](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "integers": (
+            "[integers](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "boolean": (
+            "[boolean](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "booleans": (
+            "[booleans](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+    }
+    string = dict_.get(string, string)
+    return string
+
+
+def _resolve_metadata_type(definition):
+    """Generate string of metadata type from dictionary."""
+    if "type" in definition.keys():
+        string = _get_link(definition["type"])
+
+        if (
+            ("enum" in definition.keys())
+            and (len(definition["enum"]) == 1)
+            and (definition["enum"][0] == "n/a")
+        ):
```

Comment:
Assuming that we're expecting a list not a tuple...

```suggestion
        if defintion.get("enum") == ["n/a"]:
```